.github/workflows/maven-package.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        java: [ '11', '17' ]
+        java: [ '21' ]
```

Comment:
This would currently need to be `21-ea` I think.

Reply:
Good catch. Thanks! Fixed. :-)